imageio/imageio-pict/src/main/java/com/twelvemonkeys/imageio/plugins/pict/PICTImageReader.java

```diff
@@ -2074,9 +2072,15 @@ else if (packType == 4) {
                 }
 
                 if (DEBUG) {
-                    System.out.print("Line " + scanline + ", byteCount: " + packedBytesCount);
-                    System.out.print(" dstBytes: " + dstBytes.length);
-                    System.out.println();
+                    if (scanline<5 || scanline>=srcRect.height-5) {
```

Comment:
Ok, one more time. 😀 

```suggestion
                    if (scanline < 5 || scanline >= srcRect.height - 5) {
```